### PR TITLE
Spatial audio updates for windows

### PR DESCRIPTION
**Pull request by hussain-khalid**

### CODE REVIEW

File: shared/video-sdk/develop/spatial-audio/project-implementation/windows.mdx [hunk 1]
```diff
@@ -99,39 +91,32 @@ To implement these features in your <Vpl k="CLIENT" />, take the following steps
 
 1. **Set the spatial position of a remote user**
 
-    To update the spatial position of a remote user, define the `RemoteVoicePositionInfo` and call `updateRemotePosition`. To implement this workflow, in **Dialog Editor**, double-click **Spatial Audio**. **Dialog Editor** automatically creates and opens an event listener for you. Add the following code to the event listener you just created:
+    When a user moves the slider, you create a `RemoteVoicePositionInfo` object and specify information about the position and orientation of the remote user audio stream. Then, you call `updateRemotePosition` with the `RemoteVoicePositionInfo` object and `remoteUId` to update the remote user spatial audio position. To implement this workflow, in **Dialog Editor**, double-click the slider control. **Dialog Editor** automatically set up an event listener for you. Add the following code to the event listener you just created:
 
         ```cpp
+        int value = distnaceBar->GetPos();
```
Comment (Pankajg123) [inline shared/video-sdk/develop/spatial-audio/project-implementation/windows.mdx:97]: distanceBar undefined.
it should be initialize before using :

CSliderCtrl* distnaceBar = reinterpret_cast<CSliderCtrl*>(GetDlgItem(IDC_SLIDER1));

where  `IDC_SLIDER1` is the diagram id for taken slider.

File: shared/video-sdk/develop/spatial-audio/project-implementation/windows.mdx [hunk 2]
```diff
@@ -39,7 +39,7 @@ To implement these features in your <Vpl k="CLIENT" />, take the following steps
     You create an instance of `ILocalSpatialAudioEngine` to configure spatial audio and set self and remote user positions. In `AgoraImplementationDlg.h`, add the following to `CAgoraImplementationDlg`:
 
     ```cpp
-    ILocalSpatialAudioEngine localSpatial;
+    ILocalSpatialAudioEngine* localSpatial;
```
Comment (Pankajg123) [inline shared/video-sdk/develop/spatial-audio/project-implementation/windows.mdx:42]: I suppose you mean the pointer variable , as you can not instantiate the interface (as it contain pure virtual function)

**Comment by billy-the-fish** [comment]
@hussain-khalid, can you update following these comments please.

**Comment by Pankajg123** [comment]
> @hussain-khalid, can you update following these comments please.

I have already updated the code where it was needed . Just put the comment to get confirmation from @hussain-khalid . Thanks

**Review by Pankajg123 — APPROVED** [review]

**Pull request merged**